src/software/ai/hl/stp/play/defense_play.cpp

```diff
@@ -74,8 +70,9 @@ void DefensePlay::getNextTactics(TacticCoroutine::push_type &yield, const World
         // extra friendly robots, have them perform a reasonable default defensive tactic
         if (enemy_threats.size() > 0)
         {
-            defense_shadow_enemy_tactic->updateControlParams(enemy_threats.at(1));
-            result[0].emplace_back(defense_shadow_enemy_tactic);
+            std::get<0>(shadow_enemy_tactics)
+                ->updateControlParams(enemy_threats.at(0), ROBOT_MAX_RADIUS_METERS * 3);
```

Comment:
```suggestion
                ->updateControlParams(enemy_threats.at(1), ROBOT_MAX_RADIUS_METERS * 3);
```
This doesn't make much of a difference but it does match what we had previously

Reply:
Does this make sense though? Shouldn't we assign the first enemy threat to the first shadow enemy tactic? Like if enemy threat size was 1 I wouldn't want to assign it to the 2nd shadow enemy.

Reply:
This does because the attacker tactic is on the first enemy threat(the enemy robot with the ball). The above would then assign the first shadow enemy to the second enemy threat and the second shadow enemy to the enemy with the ball. Each enemy robot is counted as an enemy threat so there is little difference, however, my suggestion may cause a segfault so ill mark this resolved.